database.js

```diff
@@ -1,51 +1,54 @@
+// Auto-generated file by https://github.com/geodynamics/aspect/doc/make_cite_html.py
```

Comment:
That link doesn't work for me. Although it's clear what you mean, maybe either put the full link https://github.com/geodynamics/aspect/blob/master/doc/make_cite_html.py, or just say aspect/doc/make_cite_html.py.

Reply:
fixed